@crazy-max looks like some additional packages have to be pulled into vendor;

```

[2022-10-20T22:24:32.383Z] vendor/github.com/moby/buildkit/util/progress/logs/logs.go:13:2: cannot find package "github.com/armon/circbuf" in any of:
[2022-10-20T22:24:32.383Z] 	/go/src/github.com/docker/docker/vendor/github.com/armon/circbuf (vendor tree)
[2022-10-20T22:24:32.383Z] 	/usr/local/go/src/github.com/armon/circbuf (from $GOROOT)
[2022-10-20T22:24:32.383Z] 	/go/src/github.com/armon/circbuf (from $GOPATH)
```

--- a/vendor.conf
+++ b/vendor.conf
@@ -33,7 +33,8 @@ github.com/imdario/mergo                            1afb36080aec31e0d1528973ebe6
 golang.org/x/sync                                   cd5d95a43a6e21273425c7ae415d3df9ea832eeb
 
 # buildkit
-github.com/moby/buildkit                            c014937225cba29cfb1d5161fd134316c0e9bdaa # v0.8.3-31-gc0149372
+github.com/armon/circbuf                            5111143e8da2e98b4ea6a8f32b9065ea1821c191
+github.com/moby/buildkit                            eeb7b65ab7d651770a5ec52a06ea7c96eb97a249 # v0.8.4-0.20221020190723-eeb7b65ab7d6
 github.com/tonistiigi/fsutil                        0834f99b7b85462efb69b4f571a4fa3ca7da5ac9
 github.com/tonistiigi/units                         6950e57a87eaf136bbe44ef2ec8e75b9e3569de2
 github.com/grpc-ecosystem/grpc-opentracing          8e809c8a86450a29b90dcc9efbf062d0fe6d9746

